### DISCUSSION TITLE
Print comments together with the recipe - how to do it?

### DISCUSSION THREAD

**Discussion by sg17843** [TandoorRecipes/recipes#1443] [Q&A]
Hi,

since I've only been using Tandoor for a few days, maybe I'm being a little silly .... I just do not make it to print my comments together with the recipe :-( How to do that please?

If this function does not exist, please add the option "... with comments" when printing.

Best regards

**Comment by vabene1111** [comment]
maybe a no print class has snuck into the comments section, i will take a look
